cmd/grapi-gen-type/di/app.go

```diff
@@ -0,0 +1,12 @@
+package di
+
+import (
+	"github.com/izumin5210/grapi/pkg/gencmd"
+	"github.com/izumin5210/grapi/pkg/protoc"
+)
+
+type CreateAppFunc func(*gencmd.Command) (*App, error)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CreateAppFunc should have comment or be unexported